libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/posts/rest_v12_sites_id_posts_new.json

```
@@ -6,20 +6,13 @@
             "context": {
                 "equalTo": "edit"
             }
-        },
-        "bodyPatterns": [
-            {
-                "equalToJson": "{\"format\":\"standard\",\"title\":\"Title\",\"content\":\"Content\",\"status\":\"publish\"}",
-                "ignoreArrayOrder": true,
-                "ignoreExtraElements": true
-            }
-        ]
+        }
     },
     "response": {
         "status": 200,
         "fixedDelayMilliseconds": 800,
```

AS mentioned in https://github.com/wordpress-mobile/WordPress-Android/pull/9795/files#r283760239, this request is returning so quickly that the test fails sometimes. Here is a workaround:

```suggestion
        "fixedDelayMilliseconds": 1000,
```